pkgs/by-name/mc/mcat/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
```

Comment:
Perhaps you could add `withChromium` and `withFfmpeg` options here? Fetching these programs on demand doesn't seem to work on NixOS.